### PR TITLE
Docker pipeline guide

### PR DESCRIPTION
**Pull request by philippmwirth**
# Docker pipeline guide

Closes [#934](https://github.com/lightly-ai/lightly-core/issues/934).

This is a first draft, feedback is very welcome. Here's the pdf (it looks much better if built locally)
[dagster-lightly-pipeline-draft.pdf](https://github.com/lightly-ai/lightly/files/6980452/dagster-lightly-pipeline-draft.pdf)

### CODE REVIEW

File: docs/source/docker/integration/dagster_aws.rst [hunk 1]
```diff
@@ -0,0 +1,493 @@
...
+
+            Generates a random subfolder so the file will be stored at:
+            >>> BUCKET_NAME/RANDOM_SUBFOLDER/basefilename.mp4
+            
+            """
+
+            # upload file to lightly-aws-bucket/input_dir/RANDOM_STRING/basename.mp4
+            object_name = os.path.join(
+                'input_dir',
+                self.random_subfolder(),
```
Comment (MalteEbner) [inline docs/source/docker/integration/dagster_aws.rst:260]: Why should it generate a random subfolder? Why not call the subfolder always `lightly_data_input` or similar?

**Review by MalteEbner — APPROVED** [review]
Wow, I like the tutorial a lot, easy to follow,, but in-detail enough to get around the hard bites

**Comment by MalteEbner** [comment]
Will you also include a link to the repo, so that users can directly download all code in well-named files?

File: docs/source/docker/integration/dagster_aws.rst (hunk 1 above)
Comment (philippmwirth) [reply docs/source/docker/integration/dagster_aws.rst:260]: Because we require one input folder per video for this workflow.

**Comment by philippmwirth** [comment]
> 
> 
> Will you also include a link to the repo, so that users can directly download all code in well-named files?

I could do that but so far, the repo is private.

**Pull request merged**